validator/testcases/javascript/predefinedentities.py

```diff
@@ -64,6 +101,12 @@
 ]
 
 
+def is_shared_scope(t, r=None, rn=None):
```

Comment:
This seems like it is moving the conditional from `is_shared_scope() and readonly_value` to when `readonly_value` is set which probably makes sense. I don't see `readonly` being set to anything other than this though but maybe I'm missing something?